test/upgrade/before/setup_test.go

```diff
@@ -113,12 +115,27 @@ var (
 	ignoreServicePortNodePortProtocolField = cmpopts.IgnoreFields(corev1.ServicePort{}, "NodePort", "Protocol")
 	ignoreRPSClusterNameField              = cmpopts.IgnoreFields(placementv1beta1.ResourcePlacementStatus{}, "ClusterName")
 
+	// Since Fleet agents v0.14.0 a minor reason string change was applied on the hub side that
+	// affects CRP availability status reportings in the resource placement section when untrackable
+	// resources are involved. This transformer is added to ensure that the compatibility test specs
+	// can handle this string change smoothly.
+	//
+	// Note that the aforementioned change is hub side exclusive and is for informational purposes only.
+	availableDueToUntrackableResCondAcyclicTransformer = cmpopts.AcyclicTransformer("AvailableDueToUntrackableResCond", func(cond metav1.Condition) metav1.Condition {
```

Comment:
when will this be removed?

Reply:
Hi Ryan! This can be removed once the [last tag - 2] is >= 0.14.0.